packages/server/test/index.ts

```diff
@@ -39,6 +37,18 @@ async function createStoreController () {
   })
 }
 
+async function testErrorFormat (t: Test, err: GotError, expectedCode: number) {
```

Comment:
you don't necessarily need to refactor in tests.
It is ok if you just copy-paste some code into each testcase.